### ISSUE TITLE
Possible reading of hidden buttons

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/models-of-the-hydrogen-atom#155]
**Device**
Samsung Laptop
**OS**
Win 11
**Browser**
Chrome + JAWS
**Problem Description**
For https://github.com/phetsims/qa/issues/1214. 
I had an instance of the screen reader reading out the three buttons on the spectrometer graph when the box was closed and the buttons hidden on the second screen. However, immediately after I was unable to reproduce this and I haven't been able to reproduce since, even after refreshing the page. I'm starting to question if I actually saw it, it's possible I misheard as I tabbed through. But just in case I'm creating an issue.

**Comment by jessegreenberg** [comment]
I tested for this but could not get it to happen with Windows 11 Chrome and NVDA 2024.4.

**Comment by pixelzoom** [comment]
@Nancy-Salpepi would you mind trying to reproduce what @KatieWoe described? If you cannot reproduce, then please close this issue.

**Comment by arouinfar** [comment]
I don't think @Nancy-Salpepi has access to JAWS, as it is a Windows-only screen reader. Who else on the team has access to JAWS? Please re-assign to them to see if they can reproduce.

>  However, immediately after I was unable to reproduce this and I haven't been able to reproduce since, even after refreshing the page. 

If no one can reproduce this bug, I recommend closing this issue.
